frontend/src/v5/ui/controls/errorMessage/errorMessage.component.tsx

```diff
@@ -14,18 +14,33 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Container, ErrorIcon, Message } from './errorMessage.styles';
```

Comment:
Are we sure we wan to use this design in the rest of the app? I agree it would be best to have just one way of display error message. But we should take in consideration if it fits all the places this is being used.

Reply:
I asked George about it and he said to replace all the old errorMessage.

I think the avatarError may be better off with the old one actually. About the others, I am not sure.
@george-hadfield ?